compiler/src/dmd/globals.d

```diff
@@ -401,12 +398,11 @@ extern (C++) struct Global
         else version (IN_LLVM)
         {
             compileEnv.vendor = "LDC";
-
-            import dmd.console : detectTerminal;
-            params.v.color = detectTerminal();
         }
+        else
+            static assert(0, "unknown vendor");
 
-        params.v.errorPrintMode = ErrorPrintMode.printErrorContext; // Enable error context globally by default
+        //params.v.errorPrintMode = ErrorPrintMode.printErrorContext; // Enable error context globally by default
```

Comment:
```suggestion
```

Is there a reason to keep this line?

Reply:
I'll take it out. I commented it out to see if it disrupted anything, but apparently not. The idea hear is that params should be initialized in one place, not scattered around.